test/languages/common-lisp_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+module SnippetExtractor
+  module Languages
+    class CommonLispTest < Minitest::Test
+      def test_full_example
+        code = <<~CODE
+          ;;;
+          ;;; Example of how to convert aribic numerals to roman numerals
+          ;;;
+          (defpackage #:roman-numerals
+            (:use #:cl)
+            (:export #:romanize))
+
+          (in-package #:roman-numerals)
```

Comment:
`in-package` could also be ignored (and I realize I could ignore it by having the pattern file contain `(in-package` but then the presented snippet would be incorrect code.